src/test/java/org/broadinstitute/hellbender/utils/UtilsUnitTest.java

```diff
@@ -428,7 +433,7 @@ public void testUnsuccessfulRegularReadableFileCheck(final File file) {
         if (file == null){
             throw new SkipException("cannot make a file unreadable (maybe you're running as root)");
         }
-        Utils.regularReadableUserFile(file);
+        Utils.canRead(file);
```

Comment:
Note also the use of "RRFCD" in the data provider, which you should change as well (lest it become a mysterious, vestigial acronym).